testapp/exam/models.py

```diff
@@ -298,6 +300,9 @@ class AnswerPaper(models.Model):
     # The questions successfully answered (a list of ids separated by '|')
     questions_answered = models.CharField(max_length=128)
 
+    # The unanswered questions (a list of ids separated by '|')
+    questions_unanswered = models.CharField(max_length=128)
```

Comment:
Why do you need a new model attribute?  All you need is a new method:
`get_unanswered_questions` which does whatever necessary.